app/views/events/_form.html.erb

```diff
@@ -147,11 +147,11 @@
       <div>
         <%= f.label :location_id, label: 'Event Location' %>
       </div>
-      <%= f.collection_select(:location_id,
-                              Location.includes(:region).available,
-                              :id,
-                              :name_with_region,
-                              {prompt: true}) %>
+      <%= f.select(:location_id,
+                   locations_for_select,
+                   include_blank: 'Please select',
```

Comment:
The `onchange` doesn't show up in the DOM because it needs to be in the fourth argument slot: changing to something like
```
<%= f.select(:location_id,
              locations_for_select,
              {include_blank: 'Please select'},
              onchange: '...' %>
```
should work.

Can you also write a feature/integration test covering this?